libraries/anvill_passes/src/BranchAnalysis.cpp

```diff
@@ -0,0 +1,432 @@
+#include <anvill/BranchAnalysis.h>
+#include <anvill/ConstraintExtractor.h>
+#include <anvill/Constraints.h>
+#include <llvm/IR/InstIterator.h>
+#include <llvm/IR/Instructions.h>
+#include <remill/BC/IntrinsicTable.h>
+
+#include <exception>
+#include <iostream>
+#include <unordered_map>
+
+namespace anvill {
+// TODO:(ian) should replace this with a generic pass over a given set of intrinsics to merge with inidrect jump passes
```

Comment:
File an issue?

Reply:
Is this still relevant?

Reply:
This was resolved by the addition of the IntrinsicPass stuff, I'll remove that comment